RELEASE_PROCESS.md

```diff
@@ -0,0 +1,41 @@
+# How to cut a release-notes-bot-action release
+
+This document is aimed at members of the team who might be cutting a release of release-notes-bot-action. It serves as a checklist that can take you through doing a release step-by-step.
+
+## Prerequisites
+
+* You must have commit access to the release-notes-bot-action repository.
+
+## Releasing
+
+Please note that this document was written with the assumption that you are using a clone of the `release-notes-bot-action` repo. You have to be using a clone rather than a fork. It is advised to your do this by making a fresh clone of the `release-notes-bot-action` repo from which you will release.
```

Comment:
```suggestion
Please note that this document was written with the assumption that you are using a clone of the `release-notes-bot-action` repo. You have to be using a clone rather than a fork. It is advised that you do this by making a fresh clone of the `release-notes-bot-action` repo from which you will release.
```